modules/bbp-core/src/main/resources/schemas/bbp/core/entity/v0.1.0.json

```diff
@@ -70,6 +73,12 @@
           "description": "The entity for which this entity is a revision of.",
           "nodeKind": "xsd:dateTime",
           "maxCount": 1
+        },
+        {
+          "path": "schema:distribution",
```

Comment:
I think we need to rework other entities as now there will be the default distribution property in the parent Entity.

Reply:
Yes. I'll do that in another iteration.